sys/net/gnrc/network_layer/sixlowpan/frag/rb/gnrc_sixlowpan_frag_rb.c

```diff
@@ -461,6 +461,16 @@ static int _rbuf_add(gnrc_netif_hdr_t *netif_hdr, gnrc_pktsnip_t *pkt,
                 else if (IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_SFR) &&
                          sixlowpan_sfr_rfrag_is(pkt->data)) {
                     entry.super->datagram_size--;
+                    /* Check, if fragment is still small enough to fit datagram size.
+                     * `offset` is 0, as this is the first fragment so it does not have to be added
+                     * here. */
+                    if (frag_size > entry.super->datagram_size) {
+                        DEBUG("6lo rfrag: fragment too big for resulting datagram, discarding datagram\n");
```

Comment:
```suggestion
                        DEBUG_PUTS("6lo rfrag: fragment too big for resulting "
                                   "datagram, discarding datagram");
```

Reply:
Isn't `printf` doing that on its own? :raised_eyebrow:

Reply:
Yes, but `DEBUG_PUTS()` will even work with small stacks, while `DEBUG()` will `puts()` an warning about the stack to small. (If that `puts()` regarding the stack too small for debugging works, a `puts()` with the debug will also work.)

The CI is unhappy about the long line. So, since you have to touch this anyway, why not also use `DEBUG_PUTS()`.

Reply:
Already done in https://github.com/RIOT-OS/RIOT/pull/19680/commits/20740af10dbb899d851ff80e361e784f2cfae207 ;-)